prow/cmd/hook/main.go

```diff
@@ -29,16 +29,21 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/sirupsen/logrus"
 
+	"github.com/ghodss/yaml"
+	"github.com/gorilla/securecookie"
+	"github.com/gorilla/sessions"
```

Comment:
Group with the imports above and not test-infra.

Reply:
done